deploy/install.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 CLI="kubectl"
-ENABLE_K8S_MONITORING="false"
+ENABLE_K8S_MONITORING="true"
```

Comment:
With this change K8s monitoring cannot be disabled anymore. Option `--enable-k8s-monitoring` can therefore be removed as well.

Reply:
Do we want to support disabling it?